stable/kong/Chart.yaml

```diff
@@ -12,5 +12,5 @@ maintainers:
 name: kong
 sources:
 - https://github.com/Kong/kong
-version: 0.5.4
+version: 0.5.5
```

Comment:
Please bump this up to 0.5.6.
We will merge in #7362 before this one.

Reply:
Done.